src/config-array.js

```diff
@@ -937,6 +951,15 @@ export class ConfigArray extends Array {
 		return this.getConfig(filePath) === undefined;
 	}
 
+	/**
+	 * Determines if the given filepath has a matching config.
+	 * @param {string} filePath The complete path of a file to check.
+	 * @returns {boolean} True if the path has a matching config, false if not.
+	 */
+	isFileConfigured(filePath) {
+		return this.getConfigOrNoMatch(filePath) !== NoMatchingConfig;
+	}
```

Comment:
If the file is ignored by ignore patterns, this returns `true` regardless of whether or not it has a matching config?

Reply:
Yes. This is technically okay, because either way the error message "File ignored because of a matching ignore pattern." does apply. But maybe the method could have a more descriptive name?

Reply:
Perhaps we could change the return type of `getConfig()`? Instead of returning `config` directly, it could return a `{ config }` object with an additional property that would indicate why is `config` undefined if that's the case.

Reply:
That's an interesting idea, it would be a breaking change though.